HeliPort/Appearance/vi.lproj/Localizable.strings

```diff
@@ -50,16 +50,16 @@
 "Wi-Fi: On" = "Wi-Fi: Bật";
 "Wi-Fi: Status unavailable" = "Wi-Fi: Không có thông tin";
 "\" Requires Password" = "\" Yêu cầu mật khẩu";
-"    IP Address: "      = "    Địa chỉ IP: ";
-"    Router: "          = "    Router: ";
-"    Internet: "        = "    Internet: ";
-"    Security: "        = "    Bảo mật: ";
-"    BSSID: "           = "    BSSID: ";
-"    Channel: "         = "    Kênh: ";
-"    Country Code: "    = "    Mã quốc gia: ";
-"    RSSI: "            = "    RSSI: ";
-"    Noise: "           = "    Noise: ";
-"    Tx Rate: "         = "    Tỷ lệ Tx: ";
-"    PHY Mode: "        = "    Dạng PHY: ";
-"    MCS Index: "       = "    Chỉ số MCS: ";
-"    NSS: "             = "    NSS: ";             
+"    IP Address: "      = "    Địa chỉ IP:";
+"    Router: "          = "    Router:";
+"    Internet: "        = "    Internet:";
+"    Security: "        = "    Bảo mật:";
+"    BSSID: "           = "    BSSID:";
+"    Channel: "         = "    Kênh:";
+"    Country Code: "    = "    Mã quốc gia:";
+"    RSSI: "            = "    RSSI:";
+"    Noise: "           = "    Noise:";
+"    Tx Rate: "         = "    Tỷ lệ Tx:";
+"    PHY Mode: "        = "    Dạng PHY:";
+"    MCS Index: "       = "    Chỉ số MCS:";
+"    NSS: "             = "    NSS:";
```

Comment:
Please don't remove necessary spaces after the colons

Reply:
I've just edited it.